standup_notes/__main__.py

```diff
@@ -124,6 +164,36 @@ def iterate_weekday(day: date, func: Callable[[date], date]) -> date:
         return iterate_weekday(next_day, func)
 
 
+def delete_notes(amount):
+    file_to_delete = []
+    if len(str(amount)) != 8:
+        print("Please enter in a valid date")
+        return
+    for file in os.listdir(STANDUP_NOTES):
+        value = int(file.split('.')[0])
+        if value < amount:
+            file_to_delete.append(file)
+    if file_to_delete:
+        print("Here are the file to be deleted")
+        for file in file_to_delete:
```

Comment:
You can use some python fanciness instead of iterating: 
print(*file_to_delete, sep='\n')

Reply:
Will do, thank you